test/common_test/free_joint_features.cc

```diff
@@ -316,6 +316,121 @@ TEST_F(FreeGroupFeaturesTest, FreeGroupSetWorldPosePrincipalAxesOffset)
   }
 }
 
+TEST_F(FreeGroupFeaturesTest, FreeGroupSetWorldPoseStaticAndFixedModel)
+{
+  const std::string modelStr = R"(
```

Comment:
nit: rename to `modelStaticStr`

Reply:
[ded1e0a](https://github.com/gazebosim/gz-physics/pull/700/commits/ded1e0a7efcdb9c61d69e917a5c242a4aa423bbc)